src/infrastructure/repositories/configsRepository.ts

```diff
@@ -33,11 +31,16 @@ export class ConfigsRepository implements IConfigsRepository {
             throw new Error('CONFIGURATIONS IT IS NOT DEFINED ON DATABASE');
         }
 
-        return {
+        const configs: Configs = {
             imageCooldownEnabled: result.imageCooldownEnabled,
             imageCooldownTime: result.imageCooldownTime,
             systemPrompt: result.systemPrompt,
             botPrefix: result.botPrefix,
+            botNumber: result.botNumber,
         }
+
+        this.configs = configs;
```

Comment:
acho que aqui ficaria legal deixar o `this.configs` private,  voltar o `fetchConfigs` pra `getConfigs` e fazer tipo:

```
if (this.configs) return this.configs
this.configs = // fetch
return this.configs
```
que aí não temos que dar fetch e dps usar a prop (com o risco de usar o configs undefined, caso esqueca de dar o fetch), sempre usamos o metodo `getConfigs`,  mas continuamos cacheando ela

Reply:
done.